mobile/lib/modules/backup/background_service/background.service.dart

```diff
@@ -0,0 +1,379 @@
+import 'dart:async';
```

Comment:
probably rename this file to `android_background_service` I think we will have the same counterpart for iOS as well

Reply:
I've read about dart/flutter platform specific plugins and usually you use the same dart file/class to communicate with both Android and iOS.